actions/lib/ansible_base.py

```diff
@@ -21,6 +22,22 @@ def __init__(self, args):
         :type args: ``list``
         """
         self.args = args[1:]
+
+        # handle multiple variables in extra_vars list
+        for i, arg in enumerate(self.args):
```

Comment:
Ideally to add this logic via a decorator same as it's done in `cmd` (https://github.com/StackStorm-Exchange/stackstorm-ansible/blob/bb86bce334f1ef5ac4c78f41872f114f16fd3fa9/actions/lib/ansible_base.py#L50) or at least extract this part into a separated function with a good naming, since the change is not so obvious.
Just a recommendation, - I can do that later in a separated PR.